src/strands/session/session_manager.py

```diff
@@ -71,3 +101,41 @@ def initialize(self, agent: "Agent", **kwargs: Any) -> None:
             agent: Agent to initialize
             **kwargs: Additional keyword arguments for future extensibility.
         """
+
+    def write_multi_agent_json(self, source: "MultiAgentBase") -> None:
```

Comment:
This methods don't seem to match the pre-existing SessionManager apis.

Can you - in the PR description - indicate what new methods are being introduced and what the parallels are? For instance, is `read_multi_agent_json` is the equivalent of `initialize` or `sync_agent`? We should also call out why (if necessary) they are named differently

Reply:
Update this signature to match sync_agent:

```
SessionManager.sync_multi_agent(self, source: "MultiAgentBase", **kwargs: Any) -> None:)
```